staging/cse/windows/kubeletfunc.tests.ps1

```diff
@@ -62,4 +62,133 @@ Describe 'Get-KubePackage' {
             Assert-MockCalled -CommandName 'DownloadFileOverHttp' -Exactly -Times 1 -ParameterFilter { $Url -eq 'https://xxx.blob.core.windows.net/kubernetes/v1.29.2/windowszip/v1.29.2-1int.zip' }
         }
     }
+}
+
+Describe 'Disable-KubeletServingCertificateRotationForTags' {
+    BeforeEach {
+        Mock Write-Log

```

Comment:
Write-Log is defined in parts/windows/windowscsehelper.ps1 and imported in this file so it seems like that we do not need to mock Write-Log